helix-term/src/ui/editor.rs

```diff
@@ -856,6 +856,52 @@ impl Component for EditorView {
                 doc.set_selection(view.id, selection);
                 EventResult::Consumed(None)
             }
+
+            Event::Mouse(MouseEvent {
+                kind: MouseEventKind::ScrollUp | MouseEventKind::ScrollDown,
+                row,
+                column,
+                ..
+            }) => {
+                let editor = &mut cxt.editor;
+                let current_view = editor.tree.focus;
+
+                let direction = match event {
+                    Event::Mouse(MouseEvent {
+                        kind: MouseEventKind::ScrollUp,
+                        ..
+                    }) => editor.config.scroll_lines.signum(),
+                    Event::Mouse(MouseEvent {
+                        kind: MouseEventKind::ScrollDown,
+                        ..
+                    }) => -editor.config.scroll_lines.signum(),
+                    _ => 0,
```

Comment:
It can be reached if user sets 0, which seems like a logical thing to do to disable scrolling.

Reply:
Right, but you're setting it to `0` here if it's an event other than scroll up/down. Since those are the only two events in this branch, it will never be anything else.

Reply:
Ah, yes, I mistakenly thought about second match. Here it's unreachable